README.md

```diff
@@ -2,17 +2,62 @@
 
 Code related to the performance testing of RH
 
+
 ## Locust
 
 For details about using Locust for load testing see https://docs.locust.io/en/stable/index.html
 
 
+### Test data
+
+In order to run Locust you'll need to populate the environment with some cases/uacs.
+This can be done by running the 'sampleGenerator' program. See the Readme in census-int-utility for notes on how to run this.
+
+Alternatively the environment may already contain some test data which can be reused, and you can get an event\_data.txt from the previous user of the environment.
+
+In any case you'll end up with a populated test\_data/event\_data.txt file which lists the data in your target environment:
+
+    $ cat test_data/event_data.txt 
+    uac,uprn,addressLine1,postcode
+    89G4NBM83RFGXW6G,200002136746,20, BN18 0ST
+    N5QQZP2WDK3LX9ZZ,100010914979,196 Paulhan Street, BL3 3DX
+    72B7TPRGM5HY7QXP,100021775714,Wych Elm, KT3 4SH
+    Y6XKSYN8ND56QWK3,100040115266,16 St Dominic Street, TR18 2DL
+    ...
+
+
 ### Run - Local
 
-Clone the repository. Change to the census-rh-performance-test directory where the repository was cloned. Run
+Firstly make sure that you have set up your environment:
+* Clone the census-rh-performance-test repository and 'cd' into it.
+* You are running Python 3.7.7
+* You have run 'pip3 install locust'. Running 'locust --version' should then return something like 'locust 1.3.2'.
+* Do a pip3 install for other dependencies. Please update this readme with a list once known.
+
+To run Locust on the command line:
 
     $ pipenv shell
-    $ locust -f ./locust_tasks/locustfile.py  --host=http://localhost:9092
+    $ 
```

Comment:
It might be worth pointing out here that there are a couple of environment variables, which need to be set after "pipenv shell" and before running locust itself, which are INSTANCE_NUM and MAX_INSTANCES. Also, although those are explained at the end of the readme, I must confess that it took me a long while to realise that these variables are now talking about locusts as such but more about what data gets used in the tests. So I would say something like this:
```suggestion
    $ #Depending on how many rows of test data you have it is necessary to tell test locust tests what section of it
    $ #you wish to use. The MAX_INSTANCES tells it how many sections you would like to split it into. And the
    $ #INSTANCE_NUM tells it which section you wish to use. For example, if you have 16 rows of data and you
    $ #want to use records 9 --> 16 then export MAX_INSTANCES=2 and export INSTANCE_NUM=2
    $export MAX_INSTANCES=2
    $export INSTANCE_NUM=2
```

Reply:
I've the instance environment variables to example.
Added short description and referred the reader to the description of these at the bottom of the readme.